website/static/js/pages/project-dashboard-page.js

```diff
@@ -231,6 +231,19 @@ $(document).ready(function () {
         var urlFilesGrid = nodeApiUrl + 'files/grid/';
         var promise = m.request({ method: 'GET', config: $osf.setXHRAuthorization, url: urlFilesGrid});
         promise.then(function (data) {
+            var newComponentElem = document.getElementById('newComponent');
+            if (window.contextVars.node.isPublic) {
+                m.mount(
+                    document.getElementById('shareButtonsPopover'),
+                    m.component(
+                        SocialShare.ShareButtonsPopover,
+                        {title: window.contextVars.node.title, url: window.location.href, type: 'link'}
+                    )
+                );
+            }
+            if (newComponentElem) {
+                m.mount(newComponentElem, AddComponentButton);
+            }
```

Comment:
This block of code is responsible for initializing the `Add Component` button on the project overview page. Previously it was run after the fangorn initialization code. Now that we removed fangorn from the overview page, fangorn initialization fails and add component initialization never happens. Moving this block of code before fangorn initialization should resolve the issue.